app/forms/select_email_form.rb

```diff
@@ -14,16 +14,25 @@ def initialize(user:, identity: nil)
   end
 
   def submit(params)
-    @selected_email_id = params[:selected_email_id]
+    @selected_email_id = params[:selected_email_id].try(:to_i)
```

Comment:
I think try will never fail here. to_i is always safe as far as I'm aware. `nil.to_i  => 0`

Reply:
There's some related discussion in Slack about this change: https://gsa-tts.slack.com/archives/C0NGESUN5/p1733256901533489

Where specifically it stems from an observation that not all parameter types are safe to `to_i`, e.g. `true`

```
> true.to_i
(irb):1:in `<main>': undefined method `to_i' for true (NoMethodError)
```